css/style.css

```diff
@@ -3,6 +3,12 @@ body {
     font-family: 'montserrat-normal', sans-serif;
 }
 
+
+.body {
+    max-width: 1440px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)